scrapy/addons.py

```diff
@@ -52,3 +52,12 @@ def load_settings(self, settings: Settings) -> None:
             },
             extra={"crawler": self.crawler},
         )
+
+    @classmethod
+    def load_spider_modules(cls, settings):
```

Comment:
Since it may be useful for other settings, I wonder if we could make it more generic, less specific to `SPIDER_MODULES`. e.g. name it `load_early_settings`, and call it right after https://github.com/scrapy/scrapy/blob/402500b164efc01257679247d3dd1628a5f90f5e/scrapy/crawler.py#L295